vvp-dossier/vvp-dossier.schema.json

```diff
@@ -0,0 +1,178 @@
+{
+  "$id": "EL3mANFohCJSfsZVc3jrqIKEZsLC-KdCC455UCOGPWGi",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "Verifiable Voice Dossier Credential",
+  "description": "Attestation credential, issued by the accountable party, that assembles all of the evidence required as per the Verifiable Voice Protocol",
+  "type": "object",
+  "credentialType": "VvpDossierCredential",
+  "version": "1.0.0",
+  "required": [
+    "v",
+    "d",
+    "i",
+    "ri",
+    "s",
+    "a",
+    "e"
+  ],
+  "properties": {
+    "v": {
+      "description": "Version",
+      "type": "string"
+    },
+    "d": {
+      "description": "Credential SAID",
+      "type": "string"
+    },
+    "i": {
+      "description": "Issuer AID",
+      "type": "string"
+    },
+    "ri": {
+      "description": "Credential status registry",
+      "type": "string"
+    },
+    "s": {
+      "description": "Schema SAID",
+      "type": "string"
+    },
+    "a": {
+      "$id": "EEMVwHco23OmIGm4k3czzLZqePejuZH9Mr11kwNnMMX3",
+      "description": "Attributes block",
+      "type": "object",
+      "properties": {
+        "d": {
+          "description": "Attributes block SAID",
+          "type": "string"
+        },
+        "dt": {
+          "description": "Issuance datetime, ISO-8601 datetime string",
+          "format": "date-time",
+          "type": "string"
+        },
+        "LEI": {
```

Comment:
I don't think this field should be present. It assumes vLEIs, which is not our intention with VVP. We should rather have an edge that points to the vetting cred, and if the vetting cred is LEI-oriented, we can get the LEI from there.

Reply:
removed LEI from attributes